src/transport/PairingSession.h

```diff
@@ -36,6 +36,28 @@ class DLL_EXPORT PairingSession
     PairingSession() {}
     virtual ~PairingSession() {}
 
+    NodeId GetPeerNodeId() const { return mPeerNodeId; }
+    void SetPeerNodeId(NodeId peerNodeId) { mPeerNodeId = peerNodeId; }
```

Comment:
Again, feels like this should be known basically at session construction/init time and should not be a public setter API.  Followup for that, please.

Reply:
Added a TODO tag.